home/tests/test_views.py

```diff
@@ -99,63 +123,74 @@ def test_get_list(self):
             revision=page.get_latest_revision(), helpful=False
         )
         page.ratings.create(revision=page.get_latest_revision(), helpful=True)
-        response = self.client.get(
-            f"{self.url}?"
-            f"{urlencode({'timestamp_gt': rating_old.timestamp.isoformat()})}"
+
+        response = api_client.get(
+            f"/api/v2/custom/ratings/?{urlencode({'timestamp_gt': rating_old.timestamp.isoformat()})}"
         )
-        self.assertEqual(
-            response.json()["results"][0],
-            ContentPageRatingSerializer(instance=rating_new).data,
+
+        assert (
+            response.json()["results"][0]
+            == ContentPageRatingSerializer(instance=rating_new).data
         )
+
         [r, _] = response.json()["results"]
         r.pop("timestamp")
-        self.assertEqual(
-            r,
-            {
-                "id": rating_new.id,
-                "comment": "",
-                "page": page.id,
-                "revision": page.get_latest_revision().id,
-                "helpful": False,
-                "data": {},
-            },
-        )
+        assert r == {
+            "id": rating_new.id,
+            "comment": "",
+            "page": page.id,
+            "revision": page.get_latest_revision().id,
+            "helpful": False,
+            "data": {},
+        }
+
+
+@pytest.mark.django_db
+class TestPageViews:
+    def create_content_page(self):
+        """
+        Helper function to create pages needed for each test.
+        """
+        home_page = HomePage.objects.first()
+        main_menu = home_page.get_children().filter(slug="main-menu").first()
+        if not main_menu:
+            main_menu = PageBuilder.build_cpi(home_page, "main-menu", "Main Menu")
+        parent = main_menu
 
+        bodies = []
 
-class PageViewsTestCase(APITestCase):
-    def setUp(self):
-        self.url = reverse("pageview-list")
+        bodies.append(WABody("default page", [WABlk("default body")]))
+
+        content_page = PageBuilder.build_cp(
+            parent=parent, slug="default-page", title="default page", bodies=bodies
+        )
+        return content_page
 
-    def test_get_list(self):
+    def test_get_list(self, api_client):
         """
         Should return the data, filtered by the querystring
```

Comment:
Again, which data? Presumably page view data, given the class name, but the docstring should be explicit about that.